### DISCUSSION TITLE
R quarto include file using subfolder not linking to ggplot2 images

### DISCUSSION THREAD

**Discussion by plukethep** [quarto-dev/quarto-cli#3318] [Q&A]
I'm trying to write a course book with some colleagues, to make things neater and stop people editing each other's work I'm putting chapters and subchapters into a folder called `chatpers` in the root of the project. These qmd files are then linked into the index.qmd file. The problem is any images created by R in these chapters (e.g. ggplot graphs) are output to` index_files\figure-html` whilst the linked qmd files create html looking for images in: `chapters/index_files/figure-html/`

How can we get the subfolder qmd files to point to the correct location for an R created image?

See more details here: https://stackoverflow.com/questions/74433972/r-quarto-include-file-not-linking-to-ggplot2-images

**Comment by mcanouil** [comment]
Could you provide a working reproducible example, such as a repository or a set of files, with how you call Quarto (i.e., CLI, CLI with some custom parameters, RStudio, VSCode, ...)? I think your issue is more about your setup rather than a bug with Quarto in this case.

**Reply by plukethep** [reply]
thanks, I'm using RStudio, no custom parameters. I'll try and put something more reproducible together
